Makefile

```diff
@@ -113,16 +112,33 @@ srpm: .srpm-done
 
 rpm: .rpm-done
 
-ubuntu-trusty:
-	cp packaging/ubuntu-trusty/ecs.conf ecs.conf
-	tar -czf ./amazon-ecs-init_${VERSION}.orig.tar.gz ecs-init ecs.conf scripts README.md
+ARCH:=$(shell uname -m)
+ifeq (${ARCH},x86_64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-v${VERSION}.tar
+else ifeq (${ARCH},aarch64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-arm64-v${VERSION}.tar
+endif
+
+/tmp/ecs-agent-version:
+	echo "${VERSION}" >/tmp/ecs-agent-version
+
+/tmp/ecs-agent.tar: /tmp/ecs-agent-version
+	curl -o /tmp/ecs-agent.tar ${AGENT_URL}
+
+.PHONY: deb
+deb: .deb-done
+.deb-done: /tmp/ecs-agent.tar
+	./scripts/update-version.sh
+	cp packaging/generic-deb/ecs.service ecs.service
+	tar -czf ./amazon-ecs-init_${VERSION}.orig.tar.gz ecs-init ecs.service scripts README.md
 	mkdir -p BUILDROOT
-	cp -r packaging/ubuntu-trusty/debian BUILDROOT/debian
+	cp -r packaging/generic-deb/debian BUILDROOT/debian
 	cp -r ecs-init BUILDROOT
-	cp packaging/ubuntu-trusty/ecs.conf BUILDROOT
+	cp packaging/generic-deb/ecs.service BUILDROOT
```

Comment:
You should copy to `BUILDROOT/debian`, where `dh_installsystemd` will pick it up. I described this more in another comment.

Reply:
done